### ISSUE TITLE
 How to deal with overfitting and underfitting for convolutional neural networks?

### ISSUE THREAD

**Issue by WatershedXiaolan** [WatershedXiaolan/Xiaolan-s-Gitbook#176]
### Overfitting
- Add more data.
- Use data augmentation.
- Use architectures that generalize well.
- Add regularization (mostly dropout, L1/L2 regularization are also possible)
- Reduce architecture complexity.
- Train less 

### Underfitting 
- Use more complex model 
- Remove regularization 
- Train more
